.pre-commit-config.yaml

```diff
@@ -72,12 +72,6 @@ repos:
         entry: .hooks/go-copyright.sh
         files: '\.go$'
 
-      - id: go-unit-tests
-        name: Go unit tests
-        language: script
```

Comment:
i can only run pre-commit in the container (all my attempts to install it outside the container failed in the past) and this is prohibitively slow in the container, so removing since it appears redundant with `tests.yaml` workflow which we also trigger automatically on PRs. This preserves my ability to run `pre-commit run  --all-files` inside the container within a reasonable time prior to pushing PRs.

Reply:
Problem with removing this is folks won't automatically get feedback via go tests when they run pre-commit. 

In terms of the container - what is the hardware you have allocated? Have you tried bumping up the available resources?